packages/components/src/components/bal-tab-slider/bal-tab-slider.tsx

```diff
@@ -0,0 +1,112 @@
+import { Component, h, ComponentInterface, Host, Element, State, Listen } from '@stencil/core'
+import { BEM } from '../../utils/bem'
+import { observeItems } from '../../utils/observer'
+
+@Component({
+  tag: 'bal-tab-slider',
```

Comment:
- [ ] could we call this component just slider

then we will merge the other sliders later into this one.

- product slider
- image slider

Reply:
I really thought about this, but I saw another component with the same name and I wanted to avoid confusions